meson.build

```diff
@@ -186,7 +189,9 @@ if buildExec.disabled() == false
 
    if host_machine.system() == 'windows'
       windows = import('windows')
-      naev_source += windows.compile_resources('extras/resource.rc')
+      icon = configure_file(input: 'extras/logos/logo.ico', output: 'logo.ico', copy: true)
```

Comment:
Does this really need to be copied to the build directory? Does a files() object not work?